### PR TITLE
Maint: Grant access to replay function from databroker.replay

### PR DESCRIPTION
**Pull request by maffettone**
Previously this function was not importable.

If this was hidden on purpose, please feel free to close this request. 

## Description

Made it importable and removed one layer of indirection in the `__init__.py`

## Motivation and Context

Seems like a lovely function to not be able to import.

## How Has This Been Tested?

`from databroker.replay import replay`

### CODE REVIEW

File: databroker/replay/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .replay import replay
```
Comment (mrakitin) [inline databroker/replay/__init__.py]:
```suggestion
from .replay import replay  # noqa F401
```

**Review by mrakitin — COMMENTED** [review]
Looks good to me. Let's make the linter happy with my suggestion.

**Review by mrakitin — APPROVED** [review]

File: databroker/replay/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from .replay import replay  # noqa F401
```
Comment (tacaswell) [inline databroker/replay/__init__.py]:
```suggestion

```

Doing this makes the underlying module inaccessible due to the shadowing.  Can we accept the extra layer or rename `replay.py` -> `__init__.py` ?

Comment (maffettone) [reply databroker/replay/__init__.py]: Somewhat on purpose (based on my interpretation of the design). Nothing else in that module is meant to be imported (private and main functions), and the main function can still be accessed by running the file or adding an entry point if needed. 

I have no qualms about changing it though if recommended style.

**Review by mrakitin — APPROVED** [review]
Let's merge it! Any objections, @tacaswell, @maffettone?

**Review by danielballan — CHANGES_REQUESTED** [review]
Making a module in accessible is a blocker from my POV. Even if it's internal, it still should be importable for things like testing.

Could rename `.replay` to `._replay` or something else.

**Comment by maffettone** [comment]
Good point. I followed the initial suggestion to place the importable function into the `__init__`, then renamed the main script as a script to remove confusion.

**Review by danielballan — APPROVED** [review]

**Pull request merged**